docs/_guide/rendering.md

```diff
@@ -84,3 +84,27 @@ class HelloWorldElement extends HTMLElement {
 ```
 
 Here, instead of declaring our template in HTML, we can do so in JS, and achieve exactly the same effect. We aren't using `@targets` in this example, as there is a more direct way to handle the data; re-calling `update()` will efficiently update only the parts that change.
+
+The same effect could be achieved using `@attr` via:
+
+```
+import { attr, controller } from "@github/catalyst"
+import { html, render } from "@github/jtml"
+
+@controller
+class HelloWorldElement extends HTMLElement {
+  @attr name = 'World'
+
+  connectedCallback() {
+    this.attachShadow({mode: 'open'})
+  }
+
+  attributeChangedCallback() {
```

Comment:
```suggestion
  attributeChangedCallback(name: string) {
    if (name !== 'name') return
```

would we want this here too?

Reply:
This isn't necessary as the only attribute changes which fire `attributeChangedCallback` will be those with the `@attr` decorator or those in `observedAttributes` - in other words in this example `name` is the only attribute that triggers `attributeChangedCallback`.